civihr_default_theme/scss/components/_panel.scss

```diff
@@ -191,15 +193,18 @@ form {
 .chr_panel__toolbar__filter {
 
   .btn.input-group-addon {
+    border: 0;
+    box-shadow: none;
     position: absolute;
-    top: 4px;
-    right: 2px;
-    padding: 6px;
+    right: 0;
+    padding: 0;
     font-size: 1.1rem;
-    line-height: 1;
+    line-height: 34px;
     background: none;
     color: $crm-color-neutral;
     cursor: default;
+    z-index: 4;
```

Comment:
Why 4 and not 5 or 12?

Reply:
![tasks___civihrnew2](https://user-images.githubusercontent.com/5058867/35271843-584f1dca-0059-11e8-80e1-de7a33caab69.png)
Because the input field has `z-index` of 3 in active state, so 4 is necessary so that the search icon gets displayed.